tasks-schema.json

```diff
@@ -356,24 +466,175 @@
+                              "type": ["array", "null"],
+                              "items": {
+                                "type": "integer",
+                                "minimum": 1
+                              }
+                            },
+                            "optional": {
+                              "description": "Array of valid but not required unique sample indexes. Can be null if all sample indexes are required and are specified in the required property.",
+                              "type": ["array", "null"],
+                              "items": {
+                                "type": "integer",
+                                "minimum": 1
+                              }
+                            }
+                          },
+                          "required": [
+                            "required",
+                            "optional"
+                          ]
+                        },
+                        "value": {
+                          "type": "object",
+                          "description": "Object defining valid values of samples from the predictive distribution for a given sample index.  Depending on the Hub specification, samples with the same sample index (specified by the type_id) may be assumed to correspond to a joint distribution across multiple levels of the task id variables. See Hub documentation for details.",
+                          "properties": {
+                            "type": {
+                              "description": "Data type of sample value from the predictive distribution.",
+                              "example": "double",
+                              "enum": [
+                                "numeric", "integer", "double"
+                              ]
+                            },
+                            "minimum": {
+                              "description": "The minimum inclusive valid sample value from the predictive distribution",
+                              "example": 0,
+                              "type": "number",
+                              "minimum": 0,
+                              "maximum": 1
+                            },
+                            "maximum": {
+                              "description": "The maximum inclusive valid sample value from the predictive distribution",
+                              "example": 1,
+                              "type": "number",
+                              "minimum": 0,
+                              "maximum": 1
+                            }
+                          },
+                          "required": [
+                            "type",
+                            "minimum",
+                            "maximum"
```

Comment:
not sure we need to require a minimum and/or maximum to be specified?
```suggestion
```

Reply:
I'm tempted to say that if we decide to hardcode minima and maxima of probability fields, we should make them a requirement to ensure the standard gets propagated into the hub configs?

Reply:
The idea of using a constant here is that when folks write their own tasks.json, there must be a minimum and maximum property in the relevant value objects and they must be 0 & 1 respectively. Now that I think of it, I think I will add that to the description so the expectation in clear.